pkg/config/model/config_overrides.go

```diff
@@ -3,9 +3,12 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2016-present Datadog, Inc.
 
-package config
+package model
 
-var overrideVars = make(map[string]interface{})
+var (
+	overrideVars  = make(map[string]interface{})
+	overrideFuncs = make([]func(Config), 0)
```

Comment:
Do you really want to redefine that here ? Or is it supposed to replace the definition from `pkg/config/config.go:110` (which you did not remove)

Reply:
I think we need this here, but should remove the one in `pkg/config/config.go`

Reply:
done. removed one in config.go